_posts/2021-06-30-gsoc'21-projects.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "JabRef GSoC’21 Projects"
+id: gsoc'21-projects
+author: JabRef GSoC Mentee
+bg: jabref-font
+color: white
+---
+
+Hey everyone!. Today's blog is about the projects selected for GSoC and a little sneak peek of the projects.
+
+
+## Microsoft Word Integration
+We are working on a Word add-in that allows users to select and insert citations 
+directly from the Word document. The JabRef Word Add-in can be used without having your reference manager open 
+or even installed. All you need is your jabref account. Once you sign in to JabRef Word Addin, your JabRef 
```

Comment:
As users don't have a JabRef account now, this is a new concept and probably needs a bit more gentle introduction; or just leave it out completely.